tcms/testruns/templates/testruns/get.html

```diff
@@ -262,6 +264,47 @@ <h5><span class="test-executions-count"></span> Results</h5>
     </div> <!-- /row -->
 </div>
 
+<div class="modal fade" id="add-link-modal" tabindex="-1" role="dialog" aria-labelledby="add-hyperlink-modal-title" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true" aria-label="Close">
+          <span class="pficon pficon-close"></span>
+        </button>
+        <h4 class="modal-title" id="add-hyperlink-modal-title">Add hyperlink</h4>
+      </div>
+      <form class="form-horizontal add-hyperlink-form">
+        <div class="modal-body">
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="{{ link_form.url.id_for_label }}">URL</label>
+            <div class="col-sm-9">
+                <input type="url" name="{{ link_form.url.name }}" maxlength="{{ link_form.url.field.max_length }}" {% if link_form.url.field.required %}required{% endif %} id="{{ link_form.url.id_for_label }}" class="form-control">
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="{{ link_form.name.id_for_label }}">Name</label>
+            <div class="col-sm-9">
+                <input type="text" name="{{ link_form.name.name }}" maxlength="{{ link_form.name.field.max_length }}" {% if link_form.name.field.required %}required{% endif %} id="{{ link_form.name.id_for_label }}" class="form-control">
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="defectCheckbox">Is a defect</label>
+            <input class="col-sm-9 bootstrap-switch" type="checkbox" id="defectCheckbox"/>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="autoUpdateCheckbox">Auto-update issue tracker</label>
+            <input class="col-sm-9 bootstrap-switch" type="checkbox" id="autoUpdateCheckbox"/>
```

Comment:
Use data-on-text/data-off-text attributes with a Yes/No, see telemetry/templates/telemetry/testing/status-matrix.html

Reply:
we don't use that attribute anywhere, because of visual consistency of the buttons. and I don't see a switch in Status matrix page.

Reply:
We now do since v8.5.

![Снимка на 2020-08-16 00-25-34](https://user-images.githubusercontent.com/1002300/90321962-2ab6e700-df57-11ea-83eb-90801eaedc15.png)